docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-1a.md

```diff
@@ -8,7 +8,12 @@ here.
 ```shell
 kubectl ws root:espw
 mailbox-controller -v=2 &
-sleep 60
+# wait until workspaces for both mailboxes show up
+sleep 10
+kubectl ws root
+while [ $(kubectl ws tree | grep "── .\+$" | wc -l) -ne 6 ]; do
```

Comment:
It would be more robust to count the number of lines that have the distinctive "-mb-" in them.

Reply:
I didn't realize this was a guaranteed feature, it's certainly preferable. I'll make the change